apps/cowswap-frontend/src/modules/bridge/components/BridgeRouteBreakdown.tsx

```diff
@@ -109,10 +113,97 @@ export interface BridgeRouteBreakdownProps {
 
   // Display options
   hideBridgeFlowFiatAmount?: boolean
+
+  // Accordion functionality
+  /**
+   * Whether the component should be collapsible/expandable.
+   * When true, clicking the header will toggle between collapsed (header only) and expanded (full breakdown) views.
+   */
+  isCollapsible?: boolean
+
+  /**
+   * Whether the component is currently expanded.
+   * - When true: shows the full route breakdown
+   * - When false and isCollapsible is true: shows only the header
+   * Only relevant when isCollapsible is true.
+   */
+  isExpanded?: boolean
+
+  /**
+   * Callback function invoked when the header is clicked in collapsible mode.
+   * The parent component should use this to toggle the isExpanded state.
+   */
+  onExpandToggle?: () => void
 }
 
+// Toggle arrow component for the collapsible header
+const ToggleArrow = styled.div<{ isOpen: boolean }>`
```

Comment:
Styled elements should be in `styled.ts`

Reply:
Addressed